kernel-modules/support-packages/04-create-support-packages.sh

```diff
@@ -48,7 +48,12 @@ for mod_ver_dir in "${MD_DIR}/module-versions"/*; do
     # For now we create *full* kernel support packages, not only deltas, in order to
     # support the slim collector use-case.
     # Remains to be clarified; we might provide more fine granular download options in the future.
-    gsutil -m cp "${COLLECTOR_MODULES_BUCKET}/${mod_ver}/*.gz" "$probe_dir"
+
+    # From 2.9.0 support packages only include downstream built drivers
+    if ! use_downstream_only "$mod_ver"; then
+        gsutil -m cp "${COLLECTOR_MODULES_BUCKET}/${mod_ver}/*.gz" "$probe_dir"
+    fi
+
```

Comment:
We might want to add a check after the next if block to prevent us from building empty support packages and issuing an error instead, WDYT?